pylabrobot/liquid_handling/utils.py

```diff
@@ -66,14 +71,14 @@ def get_tight_single_resource_liquid_op_offsets(
 
   # offsets are in absolute space.
   if resource.get_absolute_rotation().z % 180 == 0:
-    min_y = (resource.get_size_y() - channel_space) / 2
+    min_y = (resource.get_absolute_size_y() - channel_space) / 2
     if min_y < min_spacing_edge:
       raise ValueError("Resource is too small to space channels.")
     offsets = [
       Coordinate(0, min_y + i * min_spacing_between_channels, 0) for i in range(num_channels)
     ][::-1]
   elif resource.get_absolute_rotation().z % 90 == 0:
-    min_x = (resource.get_size_x() - channel_space) / 2
+    min_x = (resource.get_absolute_size_x() - channel_space) / 2
     offsets = [
```

Comment:
does it work with just `resource.get_absolute_size_y`? why is the special %90 case necessary?